### PR TITLE
reused the class names of bootstrap in the resume project

### PR DESCRIPTION
**Pull request by ahmedmelhady7**

### CODE REVIEW

File: responsive_CV/index.html [hunk 1]
```diff
@@ -4,29 +4,29 @@
 
 <head>
   <title>Ahmed Elhady's Resume</title>
```
Comment (AhmadBaracat) [inline responsive_CV/index.html]: Don't forget the viewport meta tag.

File: responsive_CV/index.html [hunk 2]
```diff
@@ -4,29 +4,29 @@
...
+  <!--<link rel="stylesheet" href="css/grid.css">-->
+  <link rel="stylesheet" href="css/bootstrap.min.css">
   <link rel="stylesheet" href="css/ui.css">
 </head>
 
 <body>
   <header>
     <div class="row">
-      <section class="intro col-5 col-12-sm">
+      <section class="intro col-lg-5 col-md-5 col-sm-12">
```
Comment (AhmadBaracat) [inline responsive_CV/index.html]: Add the `col-xs-12` to communicate your intent.

**Comment by ahmedmelhady7** [comment]
@ahmedbaracat done

**Comment by AhmadBaracat** [comment]
I think the css can be reduced and most of the spacing can be achieved using bootstrap.

**Comment by ahmedmelhady7** [comment]
@ahmedbaracat you mean what if I decided to change the color ? I would also need to change the name as well ? so instead I should choose a name that describes the semantic meaning of that html element correct maybe directly use li as the selector ?

**Comment by AhmadBaracat** [comment]
Also notice that you have generic selectors like `.col-12-sm img` which would target any image with a parent who has `.col-12-sm`. Why did you set the `width: 100%;` on the `.col-12-sm`?

**Comment by ahmedmelhady7** [comment]
@ahmedbaracat that was my own grid system names, check now

**Comment by ahmedmelhady7** [comment]
@ahmedbaracat should I merge this now after renaming the folder ?

**Review by AhmadBaracat — APPROVED** [review]

**Pull request merged**